Makefile

```diff
@@ -197,6 +197,7 @@ $(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies: package.
 		-x $@ \
 		-t node \
 		-s "$(TARGET_PLATFORM)"
+	git apply --directory $@/node_modules/lzma-native patches/cli/lzma-native-index-static-addon-require.patch
```

Comment:
Is this effectively the same as `cd`ing to `$@/node_modules/lzma-native` and then applying the patch directly in there?